lib/cartopy/mpl/patch.py

```diff
@@ -53,13 +57,16 @@ def geos_to_path(shape):
     Create a list of :class:`matplotlib.path.Path` objects that describe
     a shape.
 
+    .. deprecated:: 0.25
+       Use `shapely_to_path` instead.
+
     Parameters
     ----------
     shape
         A list, tuple or single instance of any of the following
         types: :class:`shapely.geometry.point.Point`,
         :class:`shapely.geometry.linestring.LineString`,
-        :class:`shapely.geometry.linestring.LinearRing`,
+        :class:`shapely.geometry.polygon.LinearRing`,
```

Comment:
I think these are all exported in the top-level shapely namespace so could we reference them as `shapely.LinearRing` directly now and not worry about the nested modules.